local-accounts.json

```diff
@@ -0,0 +1,62 @@
+{
```

Comment:
I guess this is the account file for cudos blast? I don't see a reason to keep it in the root directory of the project. If it is required for the tests move it there.

Reply:
I've just overlooked that one. It is not needed anywhere in the project. It is now removed.